example_app/app.ts

```diff
@@ -0,0 +1,43 @@
+import { Drash, SocketServer } from "./deps.ts";
+
+import HomeResource from "./src/resources/home_resource.ts";
+import ChatResource from "./src/resources/chat_resource.ts";
+
+const webServer = new Drash.Http.Server({
+  response_output: "application/json",
+  directory: Deno.realPathSync("."),
+  static_paths: ["/public"],
+  resources: [
+    HomeResource,
+    ChatResource,
+  ],
+  template_engine: true,
+  views_path: "./src/views",
+});
+
+webServer.run({
+  hostname: "localhost",
```

Comment:
store in config? i'm fine with it exposed too because it's just the example_app

Reply:
I don’t think the example app should have configs. It’s one less layer for developers to know about.